cumulus/client/consensus/aura/src/lib.rs

```diff
@@ -221,6 +231,18 @@ where
 			Some((validation_data.max_pov_size / 2) as usize),
 		);
 
+		// With async backing this function will be called every relay chain block.
```

Comment:
I'm still getting the error even with this patch:
```
2024-02-13 16:55:06.037 ERROR tokio-runtime-worker runtime: [Parachain] panicked at /home/alexggh/repos/polkadot-sdk/substrate/frame/aura/src/lib.rs:139:21:
Slot must increase   

2024-02-13 16:57:24.042 ERROR tokio-runtime-worker runtime: [Parachain] panicked at /home/alexggh/repos/polkadot-sdk/cumulus/pallets/parachain-system/src/lib.rs:1339:9:
no space left for the block in the unincluded segment   
```

I'm trying to find the path that would call into this function, but it doesn't seem to be used by parachains that use basic_aura::run, what am I missing ?

Reply:
Apparently there are now 100 implementations :roll_eyes:

Reply:
Fixes the issue and the parachain continues producing blocks.